lib/Assertion.js

```diff
@@ -199,8 +200,16 @@ module.exports = class Assertions {
         return value;
     }
 
+    valueAtPath(json) {
+        let jsonValue;
+        if (json) {
+            jsonValue = jsonpath.value(json, this.path);
+        }
+        return jsonValue;
```

Comment:
this looks clearer to me
return json ? jsonpath.value(json, this.path) : undefined;